src/main/SubtitleVideoArea.tsx

```diff
@@ -185,11 +186,29 @@ const VideoSelectDropdown : React.FC<{
 
   const onSubmit = () => {}
 
-  const muiTheme = createTheme({
-    palette: {
-      mode: isDarkPreferred === 'dark' ? 'dark' : 'light',
+  const muiStyle = makeStyles({
```

Comment:
This is already defined in SubtitleSelect.tsx. Can we define it in cssStyles.tsx instead to avoid duplication?